src/cswinrt/strings/additions/Windows.Storage.Streams/WindowsRuntimeBufferExtensions.cs

```diff
@@ -607,8 +618,21 @@ private static IntPtr GetPointerAtOffset(this IBuffer buffer, uint offset)
                 IntPtr buffPtr = buffer.As<IBufferByteAccess>().Buffer;
                 return new IntPtr((byte*)buffPtr + offset);
             }
+        }
+
+        private static Span<byte> GetSpanForCapacity(this IBuffer buffer, uint offset)

```

Comment:
Uses of this methods are not safe for the same reason as in the `Memory<T>` PR. It's fine to keep this since it's internal, but we need to make sure to add the appropriate GC hints so that things don't blow up.

Reply:
Added in a few places.